graphkb/genes.py

```diff
@@ -63,6 +70,43 @@ def get_oncokb_tumour_supressors(conn: GraphKBConnection) -> List[Ontology]:
     return _get_oncokb_gene_list(conn, TUMOUR_SUPPRESSIVE)
 
 
+def get_therapeutic_associated_genes(graphkb_conn: GraphKBConnection) -> List[Ontology]:
+    """Genes related to a cancer-associated statement in Graphkb."""
```

Comment:
This is from pori_ipr_python, but is easier to understand with the other gene lists in this file.